aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -349,6 +395,52 @@ resource "aws_elasticsearch_domain" "example" {
 `, randInt, instanceInt, snapshotInt)
 }
 
+func testAccESDomainConfig_WithVPCOptions(randInt int) string {
+	return fmt.Sprintf(`
+resource "aws_vpc" "foo" {
+	cidr_block = "10.10.0.0/16"
+}
+
+resource "aws_subnet" "foo" {
+	cidr_block = "10.10.1.0/24"
+	vpc_id = "${aws_vpc.foo.id}"
+}
+
+resource "aws_security_group" "foo" {
+  name = "terraform_acceptance_test_example"
+  description = "Used in the terraform acceptance tests"
+  vpc_id = "${aws_vpc.foo.id}"
+  ingress {
+    protocol = "6"
+    from_port = 80
+    to_port = 8000
+    cidr_blocks = ["10.0.0.0/8"]
+  }
+  egress {
+    protocol = "tcp"
+    from_port = 80
+    to_port = 8000
+    cidr_blocks = ["10.0.0.0/8"]
+  }
+	tags {
+		Name = "tf-acc-test"
+	}
+}
+
+resource "aws_elasticsearch_domain" "example" {
+  domain_name = "tf-test-%d"
+  ebs_options {
+    ebs_enabled = true
+    volume_size = 10
+  }
+	vpc_options {
```

Comment:
Could you fix the indentation here so that it matches the rest of the test configuration?